pyanaconda/modules/payload/live/live_os.py

```diff
@@ -54,6 +55,10 @@ def process_kickstart(self, data):
     def setup_kickstart(self, data):
         """Setup the kickstart data."""
 
+    def get_handler_path(self):
+        """Get path of this payload handle."""
+        return LIVE_OS_HANDLER.object_path
```

Comment:
I'm thinking opposite. I want to have this information in the handlers and easily accessible. The reason is again based on the multiple payloads modules in the future. It looks to me unnecessary complication to have logic in the payload to decide what will be the name of the handler instead of the handler, which knows where it should be published and how many of them are there now.

Reply:
No Anaconda modules or submodules remember their DBus path, because they don't really care about that. In general, modules can be published multiple times with different DBus paths. The main payload module will have to keep the list of created payload handlers anyway, because the client can request to change the active payload handler. The request will be based on a DBus path of the handler, so the main payload module will have to find the handler with the given path. So the dictionary is the best choice anyway.

The payload handlers are not going to have static DBus paths, so they will have to be published on a path that is given to them.

Reply:
Fixed.